src/InterfaceMoiCalculator.cc

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 Open Source Robotics Foundation
```

Comment:
```suggestion
 * Copyright 2023 Open Source Robotics Foundation
```

Reply:
Corrected in [00c1496](https://github.com/gazebosim/sdformat/pull/1304/commits/00c1496cb2e84a779b11a1e355ff3a40665b71b4)